tvrenamr/cli/core.py

```diff
@@ -69,47 +69,47 @@ def rename(config, canonical, debug, dry_run, episode,  # pylint: disable-msg=to
             _file.user_overrides(show, season, episode)
             _file.safety_check()
 
-            config = get_config(config)
+            configObject = get_config(config)
```

Comment:
Use camel_case for variable names please.

I'd also be happy with `conf` for this variable, that's a pretty standard shortening for configuration too.